styleguide/components/31-molecules/map/map.twig

```diff
@@ -1,6 +1,6 @@
-<div class="section--default">
- <div id="map-island" class="map-island--map"></div>
- <script type="text/javascript">
+<div id="map" class="map"></div>
+<script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.2.0/build/ol.js"></script>
```

Comment:
Gaat dit mogelijks geen issues geven met meerdere maps op een pagina?

Reply:
Mogelijks, maar dit ticket moet herbekeken worden later als alles van analyse / design af is. :)